var/spack/repos/builtin/packages/openspeedshop/package.py

```diff
@@ -83,8 +77,8 @@ class Openspeedshop(Package):
                          to point to target build.")
     variant('cuda', default=False,
             description="build with cuda packages included.")
-    variant('ptgf', default=False,
-            description="build with the PTGF based gui package enabled.")
+    variant('noqt3gui', default=False,
+            description="build without the build Qt3 gui package enabled.")
```

Comment:
Maybe it would be better to have a multi-valued variant so you can set `gui=qt3` or `gui=qt4`. This will prevent people from trying to use multiple at the same time. Although it might make more sense to check what version of `qt` is used and build the GUI for that version instead of using a variant.